File: app/src/main/java/com/vn/ezlearn/models/Content.java
package com.vn.ezlearn.models;

import java.util.List;

/**
 * Created by FRAMGIA\nguyen.duc.manh on 21/09/2017.
 */

public class Content {

    public Integer id;
    public String content;
    public String category_id;
    public String file_image;
    public String file_audio;
    public String status;
    public String type;
    public String parent_id;
    public String created_at;
    public String updated_at;
    public String weight;
    public String sub_question;
    public String suggest;
    public String answer_show;
    public List<Answer> answer_list;
//    public String part;
//    public String passage;
//    public String question;


//    public Content(Integer id, int type, String part, String passage, String question) {
//        this.id = id;
//        this.typeQuestion = type;
//        this.part = part;
//        this.passage = passage;
//        this.question = question;
//    }
}

File: app/src/main/java/com/vn/ezlearn/models/Content.kt
package com.vn.ezlearn.models

/**
 * Created by FRAMGIA\nguyen.duc.manh on 21/09/2017.
 */

class Content {

    var id: Int? = null
    var content: String = ""
    var category_id: String? = null
    var file_image: String? = null
    var file_audio: String? = null
    var status: String? = null
    var type: String? = null
    var parent_id: String? = null
    var created_at: String? = null
    var updated_at: String? = null
    var weight: String? = null
    var sub_question: String? = null
    var suggest: String? = null
    var answer_show: String? = null
    var answer_list: List<Answer>? = null
}
